.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "[typescript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[shellscript]": {
+    "editor.defaultFormatter": "foxundermoon.shell-format"
```

Comment:
Did you mean to add this?

Reply:
It was to format the .sh files, but it's removed now 1fb1cd9a0d9a28553d96cc287d6da573dfb0bd04